src/canisters/governance/services.ts

```diff
@@ -0,0 +1,32 @@
+import { GovernanceService } from "../../../candid/governance.idl";
+import { Command_1, ManageNeuron } from "../../../candid/governanceTypes";
+import { GovernanceError } from "../../types/governance_converters";
+import { toGovernanceError } from "./response.converters";
+
+type ManageNeuronResponse = {
+  Ok?: Command_1;
+  Err?: GovernanceError;
+};
+
+export const manageNeuron = async ({
+  request,
+  service,
+}: {
+  request: ManageNeuron;
+  service: GovernanceService;
+}): Promise<ManageNeuronResponse> => {
+  const { command } = await service.manage_neuron(request);
```

Comment:
almost the same piece of code as in `registerVote`, should we extract a function to avoid code duplication?